### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#194] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253696-00a39727-e2e3-4d2d-b147-59fed982269e.png)

**Comment by Exyss** [comment]
$\binom{10}{5} \binom{14}{9} \binom{24-5-9}{16-5-9} = \binom{10}{5} \binom{14}{9} \binom{10}{2}$
